src/app/main/lists/lists-routing.module.ts

```diff
@@ -12,11 +12,11 @@ export const routes: Routes = [
   {
     path: '', component: ListsComponent, children: [
       { path: '', redirectTo: 'list-item-text', pathMatch: 'prefix' },
-      { path: 'list-item-text', component: ListItemTextComponent },
-      { path: 'list-item-avatar', component: ListItemAvatarComponent },
-      { path: 'list-item-card', component: ListItemCardComponent },
-      { path: 'list-item-card-image', component: ListItemCardImageComponent },
-      { path: 'list-item-custom', component: ListItemCustomComponent }
+      { path: 'list-item-text', data: { oAppHeaderTitle: 'Text item list' }, component: ListItemTextComponent },
```

Comment:
Maybe it would be better to have a title like "Text items list" or "Text list item" (same for the next entries)